### DISCUSSION TITLE
[김시연] 과제 1번 제출합니다.

### DISCUSSION THREAD

**Discussion by ksy1251** [SKKUAutoLab/H-Mobility-Autonomous-Advanced-Course#36] [2024년도 과제 제출]
file:///home/siyeon1251/Pictures/Screenshots/Screenshot%20from%202024-07-11%2009-58-05.png

**Comment by Jinsun-Lee** [comment]
이미지가 확인되지 않습니다. 확인 바랍니다.

**Comment by ksy1251** [comment]
![image](https://github.com/SKKUAutoLab/H-Mobility-Autonomous-Advanced-Course/assets/120664505/136449a6-c782-425a-a6d8-183d5f322819)
